.github/workflows/docker-image.yml

```
@@ -66,7 +66,7 @@ jobs:
      - name: Build API
        working-directory: ./FountainPensNg.Server
        env:
-        ConnectionStrings__DefaultConnection: ${{ secrets.POSTGRES_CONNSTR }}
+        ConnectionStrings__FountainPens: ${{ secrets.POSTGRES_CONNSTR }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 9 but found 8

(indentation)